client/src/components/PageComponents/CalendarComponent/CalendarComponent.tsx

```diff
@@ -3,131 +3,182 @@ import Calendar from "react-calendar";
-        false, false
-    ];
-    let month = value.getMonth();
+  // 확인용 데이터
+  const isHaveTask = [
+    true,
+    false,
+    false,
+    true,
+    false,
+    true,
+    false,
+    true,
+    false,
+    true,
+    false,
+    false,
+    true,
+    false,
+    false,
+    false,
+    false,
+    false,
+    false,
+    false,
+    false,
+    false,
+    true,
+    false,
+    false,
+    true,
+    false,
+    false,
+    false,
+    false,
+  ];
 
-    const onChange = (nextValue: Date) => {
-        setValue(nextValue);
-    };
+  let month = value.getMonth();
 
-    const tileClassName = ({date, view}) => {
-        if (view === "month" && date.getMonth() === month && date.getDate()  >=1 && date.getDate() <= 30) {
-            const index = date.getDate() - 1;
-            if(isHaveTask[index]) {
-                return "have-task";
-            }
-        }
-        return "default";
-    };
+  const onChange = (nextValue: Date) => {
+    setValue(nextValue);
+  };
 
-    const handleCheckboxChange = () => {
-        setIsChecked(!isChecked);
-    };
+  const tileClassName = ({ date, view }) => {
+    if (view === "month" && date.getMonth() === month && date.getDate() >= 1 && date.getDate() <= 30) {
+      const index = date.getDate() - 1;
+      if (isHaveTask[index]) {
+        return "have-task";
+      }
+    }
+    return "default";
+  };
 
-    return (
-        <div>
-            <div className={loggoArea}>
-                <div className={textArea}>
-                    <div>
-                        현재 지역은 location 입니다.
-                        <text
-                            className={locationChangeText}
-                            onClick={() => (alert("위치 변경 모달 열기"))}
-                        >위치 변경</text>
-                    </div>
-                    <div>
-                        오늘은 날씨가 weather이므로, textObj 하시는건 어때요?
-                    </div>
-                </div>
-                <div className={loggoAreaRight}>
-                        <div
-                            className={seeAllSchedules}
-                            onClick={() => (alert("전체 일정 페이지 url 라우팅"))}
-                        >
-                            전체 일정 보기
-                        </div>
-                    <div className={iconArea}>
-                        <div className={monthText}>
-                            {month + 1}
-                        </div>
-                        <div className={iconStyle}>
-                            <BsCalendar4 />
-                        </div>
-                    </div>
-                </div>
+  const handleCheckboxChange = () => {
+    setIsChecked(!isChecked);
+  };
+
+  const gotoSchedulePage = () => {
+    navigate("/schedule");
+  };
+
+  /* 임시 코드 ................................. */
+  const openDayModal = () => {
+    navigate("/daymodal");
+  };
+
+  const gotoTodolistPage = () => {
+    navigate("/todolist");
+  };
+  /* .......................................... */
```

Comment:
아래와 같이 간략하게만 작성해주셔도 될 것 같아요~
```suggestion
  // 임시로 작성된 코드입니다.
  const openDayModal = () => {
    navigate("/daymodal");
  };

  const gotoTodolistPage = () => {
    navigate("/todolist");
  };
```